android/sdk/build.gradle

```diff
@@ -114,16 +127,32 @@ android {
 //        include 'weex-rax-extra-api.js'
 //    }
 
+    if(project.hasProperty('asfRelease')){
```

Comment:
The `if(project.hasProperty('asfRelease'))` block is not needed anymore, you can delete it.